packages/core/lib/commands/debug/inlineConfigNetwork.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function (options) {
+  const Config = require("@truffle/config");
+
+  const configObject = Config.default();
+  configObject.networks = {
```

Comment:
This is kind of a thing of principle but instead of overwriting the entire networks property, add to it like
```
configObject.networks["inline_config"] = {...
```

Reply:
this makes sense, but if we are using config `default()`, unless it is planned to change in the future, it is an empty object, right?

Reply:
You are correct, in this case it will make no functional difference. But in principle it is probably better to just add something to it rather than overwrite the entire thing.

Reply:
You can also use `Config.merge()` rather than writing to it directly if you want to be safe!  But writing to it directly should be fine here.